backend/controllers/ProfilesController.ts

```diff
@@ -1,20 +1,33 @@
 import { Express, Request, Response } from 'express';
 import { BaseController } from './BaseController';
 import { HttpStatusCode } from 'axios';
+import { Components } from '../types/openapi';
+import { AccountService } from '../services/AccountService';
+
+type Profile = Components.Schemas.Profile;
 
 export class ProfilesController extends BaseController {
   constructor(app: Express) {
     super(app, '/profiles');
   }
 
   protected initializeRoutes(): void {
-    this.router.get('/:dsnpId', this.getProfile.bind(this));
-    this.router.put('/:dsnpId', this.putProfile.bind(this));
+    this.router.get('/:msaId', this.getProfile.bind(this));
+    this.router.put('/:msaId', this.putProfile.bind(this));
   }
 
-  public getProfile(_req: Request, res: Response) {
-    return res.status(HttpStatusCode.Ok).send();
+  public async getProfile(_req: Request, res: Response) {
+    const Profile: Partial<Profile> = {
```

Comment:
nit: kinda confusing to have a const and type of the same name. I thought our convention was that only types were uppercase and values were lower case.